src/Counter/Counter.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './Counter.scss';
+
+type State = {
+  count: number;
+};
+export class Counter extends React.Component<{}, State> {
+  state = {
+    count: 0,
+  };
+
+  addOne = () => {
+    this.setState(state => ({
+      count: state.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState(state => ({
+      count: state.count + 100,
+    }));
+  };
+
+  increase = () => {
+    if (this.state.count % 5 !== 0) {
+      this.addOne();
+    } else {
+      this.addOne();
+      this.add100();
+    }
```

Comment:
```suggestion  
    this.addOne();

    if (this.state.count % 5 !== 0) {
      this.add100();
    }
```
Навіщо дублювати код?